docs/guide-ar/start-installation.md

```diff
@@ -87,12 +89,12 @@ composer create-project --prefer-dist yiisoft/yii2-app-basic basic
 <ol dir="rtl">
 <li> تثبت الملف من خلال الموقع الرسمي <a href="http://www.yiiframework.com/download/">yiiframework.com</a>.</li>
 <li> قم بفك ضغط الملف الذي تم تنزيله إلى مجلد يمكن الوصول إليه عبر الويب.</li>
-<li> قم بتعديل ملف `config / web.php` عن طريق إدخال secret key ل` cookieValidationKey`
+<li> قم بتعديل ملف <code>config/web.php</code> عن طريق إدخال secret key ل<code> cookieValidationKey</code>
 (يتم ذلك تلقائيًا إذا قمت بتثبيت ال Yii باستخدام Composer): </li>
 </ol>
 
    ```php
-   // !!! insert a secret key in the following (if it is empty) - this is required by cookie validation
+  // !!! insert a secret key in the following (if it is empty) - this is required by cookie validation
    'cookieValidationKey' => 'enter your secret key here',
    ```
```

Comment:
this code block should not be intended. in markdown that renders fine, but as you add HTML above it will not work anymore.

Reply:
Yes, that's true, I set it back to default now ..I think its change when I remove spaces between slashes.